README.md

```diff
@@ -79,25 +84,32 @@ It's likely that the Service Bus topic and subscription names will need to be am
 > Note: if you wish to run this service end to end with [Payment Responses](https://github.com/DEFRA/ffc-pay-responses), then update the `BATCH_STORAGE` environment variable to use port `10002` instead of `10007`.
 
 
-## Running the application
+# Running the File Consumer
 
 Use the convenience script, `./scripts/start`
 
-### Running tests
+# How to get an output
 
-```
-# Run all tests
-./scripts/test
+**Pre-requisite:** DAX has created a file in it's Azure File Share location.
+**Input:** A message is sent from DAX containing the name of the file to be transferred.  
+**Output:** The referenced file is copied to the configured Azure Blob Storage location and the original is deleted.  
+
+
```

Comment:
Missing

How to stop the File Consumer

The service can be stopped by using the convenience script
```
./scripts/stop
```